.github/workflows/release.yml

```diff
@@ -174,9 +174,10 @@ jobs:
           path: "./target/artifacts/vector-${{ env.VECTOR_VERSION }}-x86_64-apple-darwin.tar.gz"
 
   build-x86_64-pc-windows-msvc-packages:
-    runs-on: [self-hosted, windows, x64, general]
+    runs-on: [windows, release-builder]
```

Comment:
I think you have to update `nightly.yml` too.

Reply:
Good call.